api-reference/_hidden/dxDataGridColumn/groupCellTemplate.md

```diff
@@ -66,5 +66,7 @@ The following details should be taken into account when you use a **groupCellTem
 
 - In [fixed columns](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/columnFixing '{basewidgetpath}/Configuration/columnFixing/'), the template is initialized and rendered twice for each cell.
 
+[note] If you need to apply custom sorting for group cells, implement this and [calculateGroupValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#calculateGroupValue) properties. Refer to this <a href="https://github.com/DevExpress-Examples/DataGrid---How-to-apply-custom-sorting-to-a-grouped-column" target="_blank">Github repository</a> for an example. 
```

Comment:
It's unlikely that anybody with a sorting issue will come to this article for a solution. We should remove this paragraph.